modules/end-user-guide/partials/proc_adding-vs-code-extensions.adoc

```diff
@@ -0,0 +1,47 @@
+
+[id="adding-vs-code-extensions"]
+= Adding `VS Code` extensions
+
+To have the Microsoft Visual Studio Code - Open Source IDE automatically install chosen extensions, you have to add an `extensions.json` file to the remote Git repository that contains your project source code and that will be cloned into workspaces.
```

Comment:
I'd switch the focus from what the user can have done by the IDE to what the user can actively do to achieve this.